mission-control: revisit nix-diff

tupakkatapa opened the pull request:
Modifying the nix-diff script to retrieve only graphs (derivation trees) instead of diffs.  The differences between the derivation trees of the current and main branch can be compared by using Git.

jhvst commented:
What should we name the files? Would `nix-store-graph.txt` be good? I feel like `.log` implies a wrong thing.

tupakkatapa commented:
> What should we name the files? Would `nix-store-graph.txt` be good? I feel like `.log` implies a wrong thing.

Changed filenames to `nix-store-query.txt` and simplified output format from `--graph` to `--requisites`. Few derivations always change the hash, regardless of whether the configuration has any changes or not.

https://nixos.org/manual/nix/stable/command-ref/nix-store/query.html#queries

Ready to merge?
